### PR TITLE
New endpoints historical-climacell & realtime added

### PR DESCRIPTION
**Pull request by tatoalonso**
New endpoints have been added:

- **Climacell historica**l :  URL  --> weather/historical/climacell 
- **Realtime**                  : URL  --> weather/historical/climacell 

Test and examples for these new endpoints have been added too.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -126,6 +126,43 @@ func (c *Client) HistoricalStation(args ForecastArgs) ([]HistoricalStation, erro
...
+// to the /weather/historical/climacell endpoint, returning a slice of Weather samples on a
+// 200 response, or an ErrorResponse on a 400, 401, 403, or 500 error. You are
+// able to request  data for up to 6 hours in the past.
+//
+// Note that if the error is not due to an eror response, then the error is
+// wrapped in a pkg/errors withMessage to indicate its cause, so to work with
+// the original error, you need to call pkg/errors.Cause(). These errors are
+// things such as errors sending the request to the API, or unexpected errors
+// deserializing responses.
+func (c *Client) ClimaCellHistorical(args ForecastArgs) ([]HistoricalClimaCell, error) {
```
Comment (avivshafir) [inline client.go:139]: I think we should keep it consistent, like HistoricalStation, and use HistoricalClimaCell

File: client_test.go [hunk 2]
```diff
@@ -68,10 +68,85 @@ func historicalStationHandler() http.Handler {
 	return http.HandlerFunc(fn)
 }
 
+func historicalClimacellHandler() http.Handler {
```
Comment (avivshafir) [inline client_test.go:71]: HistoricalClimaCellHandler

**Comment by avivshafir** [comment]
Nice! @tatoalonso

File: client.go (hunk 1 above)
Comment (tatoalonso) [reply client.go:139]: you are rigth , i am going  to change this

File: client_test.go (hunk 2 above)
Comment (tatoalonso) [reply client_test.go:71]: Ooops. I will change this as well

Comment (tatoalonso) [reply client_test.go]: It´s done

**Comment by tatoalonso** [comment]
#1

File: client.go [hunk 3]
```diff
@@ -126,6 +126,43 @@ func (c *Client) HistoricalStation(args ForecastArgs) ([]HistoricalStation, erro
...
+		return nil, err
+	}
+	return f, nil
+}
+
+// RealTime returns observational data at the present time,
+// down to the minute, on successful requests
+// to the /weather/nowcast endpoint, returning a slice of Weather samples on a
+// 200 response, or an ErrorResponse on a 400, 401, 403, or 500 error. You are
+// able to request nowcast data for up to 6 hours out.
```
Comment (andyhaskell) [inline client.go:151]: This should be updated to the weather/realtime endpoint

**Pull request merged**